figures/scripts/figS7-seqcenter-barplot.R

```diff
@@ -0,0 +1,88 @@
+# S Spielman for CCDL, 2023
+# Create panel A for Figure S7 that shows how histologies (cancer groups)
+# are not balanced across RNA library preparations (polyA vs. stranded)
+
+
+#### Libraries -----------------------------------------------------------------
+
+library(tidyverse)
+library(ggpubr)
+
+#### Directories and files -----------------------------------------------------
+
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+figures_dir <- file.path(root_dir, "figures")
+output_dir <- file.path(figures_dir, "pdfs", "supp", "figS7", "panels")
+
+output_pdf <- file.path(output_dir, "RNA_library_barplot.pdf")
+
+metadata_file <- file.path(data_dir, "pbta-histologies.tsv")
+palette_file <- file.path(figures_dir, "palettes", "broad_histology_cancer_group_palette.tsv")
+binary_pal_file <- file.path(figures_dir, "palettes", "binary_color_palette.tsv")
+
+##### Read in and prepare data for plot ----------------------
+
+metadata_df <- read_tsv(metadata_file, guess_max = 10000)
+palette_df <- read_tsv(palette_file)
+binary_pal_df <- read_tsv(binary_pal_file)
+
+# Get the RNA samples and combine with `cancer_group_display` information
+library_cancer_group_df <- metadata_df %>%
+  filter(experimental_strategy == "RNA-Seq") %>%
+  # We want one row per patient here
+  mutate(dup_participant = duplicated(Kids_First_Participant_ID)) %>%
+  filter(!dup_participant) %>%
+  # Keep RNA_library and cancer group for each
+  select(RNA_library, cancer_group) %>%
+  # Cancer group display
+  inner_join(
+    select(palette_df,
+           cancer_group,
+           cancer_group_display),
+    by = "cancer_group"
+  ) %>%
+  # Remove NAs
+  drop_na(cancer_group_display) %>%
+  # Arrange cancer_group_display factor in order of total counts,
+  #  making sure "Other" is still last
+  mutate(cancer_group_display = stringr::str_wrap(cancer_group_display, 25), # make the labels fit
+         cancer_group_display = forcats::fct_infreq(cancer_group_display),
+         cancer_group_display = forcats::fct_relevel(cancer_group_display,
+                                                     "Other", after = Inf)
+  )
+
+# Prepare binary color palette
+pal_colors <- rev(binary_pal_df$hex_codes[binary_pal_df$color_names != "na_color"])
+
+###### Make the plot --------------------------
+
+rna_library_barplot <- ggplot(library_cancer_group_df) +
+  aes(x = cancer_group_display,
+      fill = RNA_library) +
+  geom_bar(color = "black",
+           size = 0.25,
+           alpha = 0.8) +
```

Comment:
I'm not sure we need/want alpha here, since there is nothing overlapping. 
```suggestion
  geom_bar(color = "black", size = 0.25) +
```